### PR TITLE
[Docker-swarm] Software Update: ELK v8.7.0

### PR DESCRIPTION
**Pull request by karun-singh**
- Moved to bitnami elk images
- Updated elk stack to v8.7.0

### CODE REVIEW

**Comment by karun-singh** [comment]
- [ ] update ES cert generation script

**Review by abhi4578 — CHANGES_REQUESTED** [review]

File: Docker-Swarm-deployment/single-node/elk/database-stack.yaml [hunk 1]
```diff
@@ -85,30 +91,34 @@ services:
...
         source: logstash-volume
-        target: /usr/share/logstash/data
+        target: /opt/bitnami/logstash/data
+    environment:
+      - BITNAMI_DEBUG=false
+      - LOGSTASH_ENABLE_MULTIPLE_PIPELINES=yes
+      - LOGSTASH_EXPOSE_API=yes
+      - LOGSTASH_API_PORT_NUMBER=9600
+      - LOGSTASH_DATA_DIR=/opt/bitnami/logstash/data
+      - LOGSTASH_HEAP_SIZE=500m
```
Comment (abhi4578) [inline Docker-Swarm-deployment/single-node/elk/database-stack.yaml:101]: Shift the ``LOGSTASH_HEAP_SIZE=500m`` to resources stack file

**Review by abhi4578 — CHANGES_REQUESTED** [review]

File: Docker-Swarm-deployment/single-node/elk/database-stack.yaml [hunk 2]
```diff
@@ -19,9 +19,16 @@ services:
         window: 10s
     networks:
       - overlay-net
+    env_file:
+      - secrets/passwords/es-password.env
+      - secrets/passwords/snapshot-credentials.env
+    environment:
+      - ELASTICSEARCH_KEYSTORE_LOCATION=/opt/bitnami/elasticsearch/config/certs/elastic-certificates.p12
```
[the remark this answers is not in the document]
Comment (karun-singh) [reply Docker-Swarm-deployment/single-node/elk/database-stack.yaml:26]: This is the tls keystore and truststore, not the same as elasticsearch.keystore. It expects either a Java keystore (jks) or a PKCS#12 file.

File: Docker-Swarm-deployment/single-node/elk/elasticsearch/elasticsearch.yaml [hunk 3]
```diff
@@ -10,3 +10,6 @@ action.auto_create_index: +.*
 xpack.security.enabled: true
 xpack.security.audit.enabled: false
 xpack.security.transport.ssl.enabled: true
+xpack.security.transport.ssl.verification_mode: certificate
+xpack.security.transport.ssl.keystore.path: /opt/bitnami/elasticsearch/config/certs/elastic-certificates.p12
```
[the remark this answers is not in the document]
Comment (karun-singh) [reply Docker-Swarm-deployment/single-node/elk/elasticsearch/elasticsearch.yaml:14]: Might be redundant and not needed after setting the location from the env variable. Will test and update.

File: Docker-Swarm-deployment/single-node/elk/database-stack.yaml [hunk 4]
```diff
@@ -19,9 +19,16 @@ services:
...
+      - ELASTICSEARCH_TRUSTSTORE_LOCATION=/opt/bitnami/elasticsearch/config/certs/elastic-certificates.p12
+      - ELASTICSEARCH_ENABLE_SECURITY=true
```
[the remark this answers is not in the document]
Comment (karun-singh) [reply Docker-Swarm-deployment/single-node/elk/database-stack.yaml:28]: Updated

File: Docker-Swarm-deployment/single-node/elk/elasticsearch/elasticsearch.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (karun-singh) [reply Docker-Swarm-deployment/single-node/elk/elasticsearch/elasticsearch.yaml]: Tested, it is redundant as the path is being set by the env variable itself. Removed.

**Review by abhi4578 — APPROVED** [review]

**Pull request merged**